pkg/pool-stable-phantom/contracts/StablePhantomPool.sol

```diff
@@ -1172,6 +1122,29 @@ contract StablePhantomPool is
         }
     }
 
+    // Translate parameters to avoid stack-too-deep issues in the constructor
+    function _extractRatesParams(NewPoolParams memory params)
+        private
+        pure
+        returns (StablePoolRates.RatesParams memory)
+    {
+        return StablePoolRates.RatesParams(params.tokens, params.rateProviders, params.tokenRateCacheDurations);
+    }
+
+    // Translate parameters to avoid stack-too-deep issues in the constructor
+    function _extractStorageParams(NewPoolParams memory params)
+        private
+        view
+        returns (StablePoolStorage.StorageParams memory)
+    {
+        return
+            StablePoolStorage.StorageParams(
+                _insertSorted(params.tokens, IERC20(this)),
+                params.rateProviders,
+                params.exemptFromYieldProtocolFeeFlags
+            );
+    }
+
```

Comment:
I'd place this right below the constructor

Reply:
I had it there, then moved it to helpers